### DISCUSSION TITLE
v4.0.0

### DISCUSSION THREAD

**Discussion by HolyWu** [HolyWu/vs-rife#30] [Announcements]
- Properly handle frame duration.
- Change default num_streams to 2.
- Put model files in a separate directory.
- Set CUDA_MODULE_LOADING environment variable to LAZY.
- Always clamp input tensor to 0.0-1.0 range.
- Remove nvFuser and CUDA Graphs.
- Change function name to lowercase.
- Add 4.7 model.
- Bump TensorRT version requirement to 8.6.1.
- Bump PyTorch version requirement to 2.0.1.
- Bump VapourSynth version requirement to R62.

<hr /><em>This discussion was created from the release <a href='https://github.com/HolyWu/vs-rife/releases/tag/v4.0.0'>v4.0.0</a>.</em>

**Comment by Samhayne** [comment]
VERY probably it’s just a stupid coincidence, and I don’t want to make anyone feel unsafe, but I still want to drop a note in case someone else had the same problem and it’s NOT just a coincidence:

One day after updating to vs-rife 4.0 something seems to have killed the VRAM of my one week old and so far well working 4090 - all channels are reported as faulty by Nvidia’s diagnostics tool “mats”.
In the mid of switching between different rife models for several minutes I suddenly got artifacts on the screen - and they came to stay.

Again: This is NOT a warning of vs-rife 4.0. The card was still pretty new and I rather guess it’s a manufacturing/material problem. I myself plan to continue using vs-rife as soon as I get a replacement card.
Still: If someone else SHOULD run into the same issue, take this as a motivation to also drop a note. :)

edit after three weeks: I tortured my old GTX 1070 for more than two weeks now with vsrife3.0 - no issues so far.
